common/src/main/scala/net/psforever/packet/game/AvatarImplantMessage.scala

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016 PSForever.net to present
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PacketHelpers, PlanetSideGamePacket}
+import scodec.Codec
+import scodec.codecs._
+
+/**
+  * An `Enumeration` of the available implants.
+  */
+object ImplantType extends Enumeration {
+  type Type = Value
+  val advanced_regen,
+      targeting,
```

Comment:
Please keep enum fields in the form of `Enum.MyValue`. Not `Enum.MY_VALUE` or `Enum.my_value`